scripts/terraform.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
It looks like all of the scripts in this project left off the training `.sh`. Might be good to keep that consistent here.